addons/im_livechat/models/res_partner.py

```diff
@@ -43,6 +43,7 @@ def _search_for_channel_invite_to_store(self, store: Store, channel):
                     # sudo: res.users.settings - operator can access other operators expertises
                     "livechat_expertise": partner.user_ids.sudo().livechat_expertise_ids.mapped("name"),
                     "livechat_languages": languages[1:],
+                    "livechat_username": partner.user_livechat_username,
```

Comment:
Also we avoid renames, the field is typically returned as `user_livechat_username` on partner.